ui/summary.go

```diff
@@ -242,6 +243,30 @@ func nonTrendMetricValueForSum(t time.Duration, timeUnit string, m *stats.Metric
 	}
 }
 
+func nonTrendMetricValueForSumJSON(t time.Duration, m *stats.Metric) map[string]interface{} {
+	data := make(map[string]interface{})
+	switch sink := m.Sink.(type) {
+	case *stats.CounterSink:
+		value := sink.Value
+		rate := 0.0
+		if t > 0 {
+			rate = value / (float64(t) / float64(time.Second))
+		}
+		data["rate"] = rate
+	case *stats.GaugeSink:
+		min := sink.Min
+		max := sink.Max
+		data["min"] = min
+		data["max"] = max
```

Comment:
I don't think you need neither `min` nor `max` as well as probably most of the variables in the other cases of this `switch`

Reply:
Fixed.